src/components/Modals/Send/views/Confirm.tsx

```diff
@@ -56,6 +65,9 @@ export const Confirm = () => {
 
   const borderColor = useColorModeValue('gray.100', 'gray.750')
 
+  // We don't want this firing -- but need it for typing
+  const handleAccountChange = () => {}
```

Comment:
Not for this PR, but leaving this here for reference: we should make a stronger discriminated union in `<AccountDropdown />` so you don't need to pass this down when `disabled` and a `defaultAccountId` is passed down, which means `<AccountDropdown />` is effectively controlled at this point thus we don't need to handle the account change